public/app/percona/check/components/AllChecksTab/ChangeCheckIntervalModal/ChangeCheckIntervalModal.messages.ts

```diff
@@ -0,0 +1,7 @@
+export const Messages = {
+  title: 'Set Interval',
+  cancel: 'Cancel',
+  save: 'Save',
+  getSuccess: (checkName: string) => `Interval changed for ${checkName}`,
+  getDescription: (checkName: string) => `Set Interval for ${checkName}`,
```

Comment:
nit: is "Interval" supposed to be capitalised?

Reply:
good catch, fixed it